src/argilla/server/models/models.py

```diff
@@ -114,6 +114,7 @@ class Record(Base):
 
     id: Mapped[UUID] = mapped_column(primary_key=True, default=uuid4)
     fields: Mapped[dict] = mapped_column(JSON, default={})
+    metadata_: Mapped[Optional[dict]] = mapped_column("metadata", JSON, nullable=True)
```

Comment:
`metadata` is a reserved attribute in sqlalchemy orm models, right?

Reply:
it is, yes